scss/_patterns_lists.scss

```diff
@@ -138,22 +144,59 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
     &.is-split .p-list__item:last-of-type {
       border-bottom: 1px solid $color-mid-light;
     }
+
+    & & {
+      margin-bottom: 0;
+      margin-left: $sp-x-large;
+      padding-top: $sph--large;
+
+      .p-list__item:last-child {
+        padding-bottom: 0;
+      }
+    }
+  }
+
+  ol.p-list--divided {
```

Comment:
Fix linting here, we need to disable targeting element in this line (we do want to target ordered lists specifically).